src/PathDrawing/PathDrawingController.cpp

```diff
@@ -0,0 +1,238 @@
+            .AddPoint(56.463931, -2.978906,altitudeMeters)
+            .AddPoint(56.463882, -2.978477,altitudeMeters)
+            .AddPoint(56.463823, -2.978093,altitudeMeters)
+            .AddPoint(56.463796, -2.978014,altitudeMeters)
+            .AddPoint(56.463773, -2.977828,altitudeMeters)
+            .AddPoint(56.463683, -2.977304,altitudeMeters)
+            .AddPoint(56.463655, -2.977026,altitudeMeters)
+            .AddPoint(56.463634, -2.976777,altitudeMeters)
+            .AddPoint(56.463636, -2.976409,altitudeMeters)
+            .AddPoint(56.463683, -2.976041,altitudeMeters)
+            .AddPoint(56.463817, -2.975528,altitudeMeters)
+            .AddPoint(56.464249, -2.974152,altitudeMeters)
+            .AddPoint(56.464311, -2.973874,altitudeMeters)
+            .AddPoint(56.464367, -2.973519,altitudeMeters)
+            .AddPoint(56.464432, -2.973333,altitudeMeters)
+            .AddPoint(56.464513, -2.973320,altitudeMeters)
+            .AddPoint(56.464578, -2.973192,altitudeMeters)
+            .AddPoint(56.464598, -2.973014,altitudeMeters)
+            .AddPoint(56.464649, -2.972897,altitudeMeters)
+            .AddPoint(56.464935, -2.971895,altitudeMeters)
+            .AddPoint(56.465076, -2.971219,altitudeMeters)
+            .AddPoint(56.465143, -2.970771,altitudeMeters)
+            .AddPoint(56.465186, -2.970261,altitudeMeters)
+            .AddPoint(56.465184, -2.969797,altitudeMeters)
+            .AddPoint(56.465160, -2.969224,altitudeMeters)
+            .AddPoint(56.465152, -2.968634,altitudeMeters)
+            .AddPoint(56.465128, -2.968031,altitudeMeters)
+            .AddPoint(56.465097, -2.967573,altitudeMeters)
+            .AddPoint(56.465056, -2.967259,altitudeMeters)
+            .AddPoint(56.464967, -2.966765,altitudeMeters)
+            .AddPoint(56.464806, -2.966072,altitudeMeters)
+            .AddPoint(56.464308, -2.964426,altitudeMeters)
+            .AddPoint(56.464328, -2.964205,altitudeMeters)
+            .AddPoint(56.464246, -2.963995,altitudeMeters)
+            .AddPoint(56.464118, -2.963946,altitudeMeters)
+            .AddPoint(56.464010, -2.964042,altitudeMeters)
+            .AddPoint(56.463774, -2.963951,altitudeMeters)
+            .AddPoint(56.463556, -2.963799,altitudeMeters)
+            .AddPoint(56.462876, -2.962969,altitudeMeters)
+            .AddPoint(56.462710, -2.962603,altitudeMeters)
+            .AddPoint(56.462633, -2.962464,altitudeMeters)
+            .AddPoint(56.462521, -2.962678,altitudeMeters)
+            .AddPoint(56.462229, -2.963078,altitudeMeters)
+            .AddPoint(56.461981, -2.963456,altitudeMeters)
+            .AddPoint(56.461686, -2.963953,altitudeMeters)
+            .AddPoint(56.461524, -2.963569,altitudeMeters)
+            .AddPoint(56.461441, -2.963602,altitudeMeters)
+            .AddPoint(56.461363, -2.963558,altitudeMeters)
+            .AddPoint(56.460882, -2.962729,altitudeMeters)
+            .FinishRoute();
```

Comment:
Presume this is temp test data. If it ends up being long lived might be worth pulling out to a MockPathData type.